### ISSUE TITLE
Make it possible to toggle checkboxes in view mode and have it save the state without entering edit mode

### ISSUE THREAD

**Issue by Raptor-TK** [dullage/flatnotes#158]
**Feature request:**
I would like to be able to go to my To do list and toggle checkboxes in my list of tasks in view mode and have it save.

**What it does now:**
Currently I can't toggle checkboxes in view mode.
I have to go to edit, view in WYSIWYG mode, check the checkbox and then click save.

**Why do I want this:**
Many people use checkboxes for quick toggling of tasks.
It is a lot of work to do 4 actions just to toggle the status of a checkbox on my todo list.

It would be faster if it could be toggled in view mode and have it saved to the note automatically.

**Comment by 766F6964** [comment]
I second this request. Manually going to edit mode and changing `[ ]` to `[x]` isn't exactly convinient.
Especially when you quickly need to check off multiple items.

**Comment by freitzzz** [comment]
Please make this a feature! I transitioned from Notion to Flatnotes but my To-do experience is still not perfect

**Comment by nvdg** [comment]
The only thing I'm really missing so far, I use it a lot for shopping lists. Other then that great app! Keep up the good work, and keep it clean!

**Comment by dullage** [comment]
This is currently the most upvoted suggestion, and so I had a look at this. Unfortunately, the viewer component of the markdown editor flatnotes uses doesn't have a method to get the content (as the editor component does). Without being able to get the content, I can't save the updated state.

I'll try and have another look for an option but thought I'd post this here as an update.

**Comment by nvdg** [comment]
Yeah I also already took a look at the viewer you are using and also thought that might get difficult to implement.

**Comment by saftl** [comment]
vote for it

**Comment by Raptor-TK** [comment]
I think the following issue in the tui.editor project is related to this issue: https://github.com/nhn/tui.editor/issues/1177
